### PR TITLE
Support Custom Stop Color

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
**Description:**

Takes custom stop color from config and passes to OTP-UI map overlay

**PR Checklist:**
- [n/a] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [n/a] Are all languages supported (Internationalization/Localization)?
- [n/a] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/components/map/default-map.tsx [hunk 1]
```diff
@@ -334,7 +336,9 @@ class DefaultMap extends Component {
               case 'park-and-ride':
                 return <ParkAndRideOverlay {...namedLayerProps} />
               case 'stops':
-                return <StopsOverlay {...namedLayerProps} />
+                return (
+                  <StopsOverlay {...namedLayerProps} stopsColor={stopsColor} />
```
Comment (miles-grant-ibigroup) [inline lib/components/map/default-map.tsx:340]: I don't think you need to explicitly pass in `stopsColor`! `namedLayerProps` already comes from the config, so you can just define it there. This PR is not needed

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
I think you may not need this PR!

File: lib/components/map/default-map.tsx (hunk 1 above)
Comment (amy-corson-ibigroup) [reply lib/components/map/default-map.tsx]: Good catch, although it's not passing through the whole `mapConfig` so I think we will need a pr! I can alternatively add the entire mapColors section of the config to the `namedLayerProps` if you think that would be better

Comment (miles-grant-ibigroup) [reply lib/components/map/default-map.tsx]: You just have to define `stopsColor` as a property of the map layer rather than as a global

Comment (amy-corson-ibigroup) [reply lib/components/map/default-map.tsx]: Gotcha, no you're right! Thanks, I'll close this and adjust accordingly!

**Comment by amy-corson-ibigroup** [comment]
Whoops! Closing as it turns out this PR is unnecessary!

**Pull request closed without merging**